src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,19 @@
+name = PropertiesConfig
+
+appenders = console
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{HH:mm:ss} %-5p %-20.20C{1} %m%n
+
+#rootLogger.level = error
+#rootLogger.level = warn
+#rootLogger.level = info
+rootLogger.level = debug
```

Comment:
Why was the XML config file removed and replaced with this hardcoded to "debug" properties file? The XML had an option to start the module with whatever log level was passed as a argument via -Dloglevel. This is now lost.

Reply:
I've reverted it back to xml